xml/System/StringNormalizationExtensions.xml

```diff
@@ -24,7 +24,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides extension methods to work with normalized strings.</summary>
```

Comment:
>normalized strings [](start = 53, length = 18)

should this be "string normalization" instead?

Reply:
I agree with @tarekgh. This should be "work with string normalization."